src/Microsoft.DotNet.Build.Tasks/FindBestConfigurations.cs

```diff
@@ -51,7 +51,12 @@ public override bool Execute()
                 }
                 else
                 {
-                    if (ignoredBuildConfigurations.Contains($"{ConfigurationFactory.NopConfigurationPrefix}{bestConfiguration}"))
+                    string targetGroup = bestConfiguration.Values[0].Value; // TargetGroup is in property value index 0
```

Comment:
While I don't know that this is wrong exactly it feels a little to specific to targetgroup/osgroup which feels a little hacky to me. Is there a more general places to do it in the Configuration class itself?